lib/grape/dsl/inside_route.rb

```diff
@@ -122,7 +122,7 @@ def status(status = nil)
         when Symbol
           raise ArgumentError, "Status code :#{status} is invalid." unless Rack::Utils::SYMBOL_TO_STATUS_CODE.keys.include?(status)
           @status = Rack::Utils.status_code(status)
-        when Fixnum
+        when integer_type
```

Comment:
Nope, Fixnum is not removed but deprecated.
This should be `Integer` because `1 === Integer` will be true. Also `Fixnum` is `Integer` on ruby-2.4.

```ruby
p Fixnum #=> Integer
```